### DISCUSSION TITLE
Can you get Github mobile app on the HUAWEI AppGallery?

### DISCUSSION THREAD

**Discussion by SakuraNeko** [community/community#17494] [Mobile] [answered]
In some countries and regions, we have difficulty accessing the Google App Store and would like to have your apps available on other app stores (e.g. HUAWEI AppGallery).

**Comment by HenSquared** [comment, marked as answer]
Hi @SakuraNeko,

Thank you for this request. 
I've opened an Issue for discussing this internally with the team.

**Reply by HenSquared** [reply]
After some discussion, we've decided not to release the Android app to any other app stores. 
Thanks again for your suggestion!

**Reply by SakuraNeko** [reply]
Thank you very much! 
If possible, hopefully your team can discuss the possibility of providing a direct download on the official website so that it doesn't need to be distributed through the app store.

**Reply by HenSquared** [reply]
@SakuraNeko, thanks for the suggestion for a direct download link. 
There are security and other technical constraints to providing a direct download of our Android app APK. We won't be able to support this for now.

I've added this suggestion to our feature requests list, we may reevaluate in the future as more users request this.

**Reply by bahamdin** [reply]
Any update about having the app release availble for public rather than that single store?

**Comment by jackace786** [comment]
Currently, the GitHub mobile app isn’t on the HUAWEI AppGallery, but you can download the APK from trusted sites like [apkgolf](https://apkgolf.net/) and install it manually!
